awx/ui_next/src/screens/Job/JobDetail/JobDetail.test.jsx

```diff
@@ -1,22 +1,60 @@
 import React from 'react';
-
 import { mountWithContexts } from '@testUtils/enzymeHelpers';
-
 import JobDetail from './JobDetail';
 
 describe('<JobDetail />', () => {
-  const mockDetails = {
-    name: 'Foo',
-  };
+  let job;
+
+  beforeEach(() => {
+    job = {
+      name: 'Foo',
+      summary_fields: {},
+    };
+  });
 
   test('initially renders succesfully', () => {
-    mountWithContexts(<JobDetail job={mockDetails} />);
+    mountWithContexts(<JobDetail job={job} />);
   });
 
   test('should display a Close button', () => {
-    const wrapper = mountWithContexts(<JobDetail job={mockDetails} />);
+    const wrapper = mountWithContexts(<JobDetail job={job} />);
 
     expect(wrapper.find('Button[aria-label="close"]').length).toBe(1);
     wrapper.unmount();
   });
+
+  test('should display details', () => {
+    job.status = 'Successful';
+    job.started = '2019-07-02T17:35:22.753817Z';
+    job.finished = '2019-07-02T17:35:34.910800Z';
+
+    const wrapper = mountWithContexts(<JobDetail job={job} />);
+    const details = wrapper.find('Detail');
+
+    function assertDetail(detail, label, value) {
+      expect(detail.prop('label')).toEqual(label);
+      expect(detail.prop('value')).toEqual(value);
+    }
+
+    assertDetail(details.at(0), 'Status', 'Successful');
```

Comment:
We could make these tests more robust by checking if the correct string is rendered in the DOM versus checking an implementation detail (props).